autogpt_platform/backend/backend/notifications/notifications.py

```diff
@@ -148,10 +187,7 @@ def _process_immediate(self, message: str) -> bool:
             event = parsed.event
             model = parsed.model
 
-            if event.recipient_email:
-                recipient_email = event.recipient_email
-            else:
-                recipient_email = self.run_and_wait(get_user_email_by_id(event.user_id))
+            recipient_email = self.run_and_wait(get_user_email_by_id(event.user_id))
             if not recipient_email:
```

Comment:
Why do I need to do this if I'm doing an admin notification ?

Reply:
This will break the admin notification on user with invalid email

Reply:
wdym? the admin user will use _process_admin_message